docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.1'
+
+services:
+    postgres:
+        image: postgres:latest
+        ports:
+            - 5432:5432
+        environment:
+            - POSTGRES_PASSWORD=superuser
+            - POSTGRES_USER=superuser
+            - POSTGRES_DB=leapchat
+        volumes:
+            - ./_docker-volumes/postgres:/var/lib/postgresql/data
+    postgrest:
+        image: begriffs/postgrest:latest
+        ports:
+            - "3000:3000"
+        environment:
+            PGRST_DB_URI: "postgres://superuser:superuser@postgres:5432/leapchat"
+        depends_on:
+            - postgres
+    adminer:
```

Comment:
Adminer is a web UI for managing SQL databases just like PHPMyAdmin. Thought it would be nice to just have in order to debug. https://www.adminer.org